scripts/tsconfig.json

```diff
@@ -0,0 +1,105 @@
+{
+  "extends": "../tsconfig.json",
+  "exclude": ["node_modules"], // cancel out the exclude from the root tsconfig
+  "compilerOptions": {
+    /* Visit https://aka.ms/tsconfig to read more about this file */
```

Comment:
Can we remove all the commented lines from this file?

Reply:
It doesn't really bother me, and it's nice to have access to all the options without having to look them up.